docs/sanitation/sanitations.bal

```diff
@@ -0,0 +1,384 @@
+    int substringStartIndex = indexOfPeriod == () ? 0 : indexOfPeriod + 1;
+    string updatedKey = schemaName.substring(substringStartIndex);
+
+    if updatedKey.endsWith("_Type") {
+        updatedKey = updatedKey.substring(0, updatedKey.length() - 5);
+    }
+
+    if updatedKey.endsWith("_Type-create") {
+        updatedKey = "Create" + updatedKey.substring(0, updatedKey.length() - 12);
+    }
+
+    if updatedKey.endsWith("_Type-update") {
+        updatedKey = "Update" + updatedKey.substring(0, updatedKey.length() - 12);
+    }
+
+    if updatedKey.endsWith("Type") {
+        updatedKey = updatedKey.substring(0, updatedKey.length() - 4);
+    }
+
+    if updatedKey.endsWith("Type-create") {
+        updatedKey = "Create" + updatedKey.substring(0, updatedKey.length() - 11);
+    }
+
+    if updatedKey.endsWith("Type-update") {
+        updatedKey = "Update" + updatedKey.substring(0, updatedKey.length() - 11);
+    }
+
+    if updatedKey.endsWith("-create") {
+        updatedKey = "Create" + updatedKey.substring(0, updatedKey.length() - 7);
+    }
+
+    if updatedKey.endsWith("-update") {
+        updatedKey = "Update" + updatedKey.substring(0, updatedKey.length() - 7);
+    }
+    return updatedKey;
+}
+
+function SanitizeWrappers(string specPath) returns error? {
+    json openAPISpec = check io:fileReadJson(specPath);
+    Specification spec = check openAPISpec.cloneWithType(Specification);
+    map<Path> paths = spec.paths;
+    foreach var [key,value] in paths.entries() {
+        if value.get != () {
+            Get getPath = value.get ?: {parameters: []};
+            map<ResponseCode> responses = getPath.responses ?: {};
+            foreach [string,ResponseCode] [_,item] in responses.entries() {
+                map<ResponseJson> content = item.content ?: {};
+                if content.hasKey("application/json"){
+                    ResponseJson app = content["application/json"] ?: {};
+                    ResponseSchema schema = app.schema ?: {};
+                    if schema.title == "Wrapper"{
+                        string suffix = key.substring(1,key.length());
+                        schema["title"] = suffix+"Wrapper";
+                    }
+                }
+            }
+        }
+        if value.post != () {
+            Post postPath = value.get ?: {parameters: []};
+            map<ResponseCode> responses = postPath.responses ?: {};
+            foreach [string,ResponseCode] [_,item] in responses.entries() {
+                map<ResponseJson> content = item.content ?: {};
+                if content.hasKey("application/json"){
+                    ResponseJson app = content["application/json"] ?: {};
+                    ResponseSchema schema = app.schema ?: {};
+                    if schema.title == "Wrapper"{
+                        string suffix = key.substring(1,key.length());
+                        schema["title"] = suffix+"Wrapper";
+                    }
+                }
```

Comment:
IMO we can move this to separate function as this is duplicated

Reply:
Please review the further improved function that supports both ODATAV2 and ODATAV4 specs. Find the changes here [Update sanitations.bal with response schema names sanitation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/4/commits/519d6db68e03dc7f3d3823d9dbf38c976ec44dc3)